tracim/tracim
# Feat/4146  update indexed content

See #4146.



## Checkpoints



**For developers**

  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**


**For testers**


**Testing method/hints**

- activate elasticsearch following the documentation in https://github.com/tracim/tracim/blob/feat/4146__update_indexed_content/backend/doc/setting.md#configure-indexing-and-search-to-use-elasticsearch-tracim-v23. You can skip the docker part and just do a `docker-compose up -d` in the backend folder before starting `./run_dev_backend.sh`.
- create contents (at root, in folders/…)
- use the search and check that the results are in line with the created documents
- edit/move the documents and check that the search results are updated.
- :warning: the contents of files is not indexed thus not searched with this configuration (as it needs an additional elasticsearch ingest service).
